README.md

```diff
@@ -107,14 +108,18 @@ fn main() {
 
 ![Benchmark result by number of spans](etc/img/benchmark-spans.svg)
 
-|                      | 1 span      | 10 spans   | 100 spans   | 1000 spans  |
-|----------------------|-------------|------------|-------------|-------------|
-| tokio-tracing        | 19x slower  | 61x slower | 124x slower | 151x slower |
-| rustracing           | 13x slower  | 26x slower | 45x slower  | 55x slower  |
-| fastrace (baseline) | 1x (0.4us)  | 1x (0.8us) | 1x (3.4us)  | 1x (27.8us) |
+|                     | 1 span     | 10 spans   | 100 spans   | 1000 spans  |
+|---------------------|------------|------------|-------------|-------------|
+| tokio-tracing       | 19x slower | 61x slower | 124x slower | 151x slower |
+| rustracing          | 13x slower | 26x slower | 45x slower  | 55x slower  |
+| fastrace (baseline) | 1x (0.4us) | 1x (0.8us) | 1x (3.4us)  | 1x (27.8us) |
 
 Detailed results are available in [etc/benchmark-result](etc/benchmark-result).
 
+## Supported Rust Versions (MSRV 1.71)
+
+Fastrace is built against the latest stable release. The minimum supported version is 1.80.1. The current Fastrace version is not guaranteed to build on Rust versions earlier than the minimum supported version.
```

Comment:
I feel like it's too aggressive to have an MSRV like this. But we can see if it works well and evaluate it as needed.

Reply:
The background is that, we don't actually define an MSRV before and it's effectively latest stable. Thus, use the current stable version is the least surprising one.

Reply:
Also reducing MSRV should not be a breaking change IIUC.

Reply:
I'm fine with this policy.